src/com/company/Matrix.java

```diff
@@ -0,0 +1,22 @@
+package com.company;
+
+import java.util.Scanner;
+
+/*
+ * @param numberOfColumns - number of columns in a matrix
+ * @param numberOfLine - number of lines in a matrix
+ * @return - creature matrix;
+ */
+public class Matrix {
```

Comment:
Class name - **Who?**
Method name - **What it does?**


Class name - **MatrixGenerator**
Method name - **create(..)**